core/src/main/kotlin/endpoints/AgentManager.kt

```diff
@@ -489,8 +489,11 @@ class AgentManager(override val kodein: Kodein) : KodeinAware {
             val handler = CoroutineExceptionHandler { _, e ->
                 logger.error(e) { "Error updating agent $agentId" }
             }
+            val settings = systemSettings.takeUnless {
+                info.agentType == AgentType.NODEJS
+            } ?: systemSettings.copy(targetHost = adminData(agentId).settings.targetHost)
```

Comment:
I think 'takeUnless' is less readable than takeIf
```suggestion
            val settings = systemSettings.takeIf {
                info.agentType != AgentType.NODEJS
            } ?: systemSettings.copy(targetHost = adminData(agentId).settings.targetHost)
```

Reply:
There is another option that we can just write: `systemSettings.copy(targetHost = adminData(agentId).settings.targetHost)`    because we use targetHost - field only in the js agent and it can't affect the settings of other agents @LikhanovMaxim  what do you think about it